samples/hubspot/README.md

```diff
@@ -0,0 +1,61 @@
+---
+title: HubSpot
+description: Simple usage of the HubSpot API
+integrations: ["hubspot"]
+categories: ["Samples"]
+---
+
+# HubSpot Sample
+
+This [AutoKitteh](https://github.com/autokitteh/autokitteh) project
+demonstrates integration with [HubSpot](https://www.hubspot.com).
+
+It sends a couple of requests to the HubSpot API, adds a contact, retrieves deals, and prints the responses back to the user.
+
+API details:
+
+- [HubSpot API](https://pypi.org/project/hubspot-api-client/)
+- [Python client library](https://github.com/HubSpot/hubspot-api-python)
+
+## How It Works
+
+1. Create a new contact in HubSpot with predefined details.
+2. Retrieve and list all deals from HubSpot, displaying their IDs and names.
+
+## Cloud Usage (Recommended)
+
+- Initialize your connection through the UI.
+2. Edit the trigger of the workflow you want to trigger, in the "TRIGGERS" tab, under the "Actions" column.
+3. Copy the provided webhook URL.
+4. Send the webhook with a curl GET or POST request, depending on the selected webhook.
+   - For create_contact_webhook:
+      ```shell
+         curl -X POST "https://api.autokitteh.cloud/webhooks/{your-webhook-slug}?email=email&firstname=name&lastname=name"
+      ```
+   - For list_deals_webhook:
+      ```shell
+         curl -X GET "https://api.autokitteh.cloud/webhooks/{your-webhook-slug}"
+      ```
+
+## Trigger Workflow
+
+Trigger the workflow using the webhook URL. Refer to the [Cloud Usage](#cloud-usage-recommended) section for detailed steps.
+
+### Steps to Retrieve the Webhook URL
```

Comment:
"Steps to" is redundant text, it dilutes the point. I'm also not sure it belong **after** step 4, because you need to do it before step 4